packages/vite-plugin/src/vite-bundle-analysis/createAsset.ts

```diff
@@ -0,0 +1,32 @@
+import { getCompressedSize, normalizePath } from "@codecov/bundler-plugin-core";
+
+interface CreateAssetOptions {
+  fileName: string;
+  source: Uint8Array | string;
+  formatString: string;
+}
+
+export const createAsset = async ({
```

Comment:
IMO just export this in `@codecov/bundler-plugin-core`. Can even namespace it under rollup, it's fine because both vite and rollup rely on it.

Reply:
Okay, yea this was the only thing i was slightly hesitant about, will do!